src/main/webapp/index.html

```diff
@@ -3,13 +3,40 @@
 
 <head>
   <meta charset="UTF-8">
-  <title>planner</title>
+  <title>TripBud</title>
+
+  <!-- Stylesheet -->
   <link rel="stylesheet" href="style.css">
+
+  <!-- Fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
+
+  <!-- JavaScript -->
   <script src="script.js"></script>
+  
+  <!-- Favicon -->
+  <link rel="icon" href="images/favicon.ico">
+
 </head>
 
 <body>
-    <p>Hi!</p>
+    <div class="container">
+
+      <h1 class="home-title">TripBud</h1>
+      <h3 class="home-subtitle">the budget trip planner</h3>
+      <div class="enter-btn-div">
+        <button class="btn login-btn">Log In</button>
+        <button class="btn signup-btn">Sign Up</button>
```

Comment:
Since we are not implementing a full login till later, what will the login button do? Do you just provide a name and get back an id from the server?

Reply:
I think for now this page create a new user by giving a string name and they'll get an id from the server. If they click log in they can give it the username and get the previously created id to fetch associated trips. I'll work on this tomorrow.